src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -722,14 +725,14 @@ public function queue_import( $args = [] ) {
 
 		if ( ! empty( $args['start'] ) ) {
 			$args['start'] = ! is_numeric( $args['start'] )
-				? Tribe__Date_Utils::maybe_format_from_datepicker( $args['start'] )
-				: date( Tribe__Date_Utils::DBDATETIMEFORMAT, $args['start'] );
+				? Dates::maybe_format_from_datepicker( $args['start'] )
+				: Dates::build_date_object( $args['start'] )->format( Dates::DBDATETIMEFORMAT)
```

Comment:
* :no_entry_sign: Expected 1 spaces before closing parenthesis; 0 found
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: c2d74943a39993049c24da672a46fdd0</sub>